src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result = days * 40;
+  } else if (days < 7) {
+    result = (days * 40) - 20;
+  } else {
+    result = (days * 40) - 50;
+  }
```

Comment:
This part can be simplified to only 2 options: first check if `days >= 7`, after it if `days >= 3`. Remember, you can immediately return result from inside `if (days >= 7) {return}` and stop the execution.